bin/aws-creds

```diff
@@ -164,15 +164,12 @@ when 'rm'
 
 when 'add-role'
   keychain = load_keychain
-  name = get_name_from_args_for_command('add-role')
-  item = get_valid_item(name)
-
-  role     = ask('         role name: ')
-  arn      = ask('          role arn: ')
+  role_name = ask('         role name: ')
+  arn       = ask('          role arn: ')
 
   keychain.generic_passwords.create(
-    label: "#{name} role #{role}",
```

Comment:
Hi @tristanmorgan, what about the other concern from #22: 

> we're removing the "namespace" of an account name with a role name, creating the potential for conflicts if different accounts have the same role name.

Do you think that's a legit concern?

Reply:
The label someone uses for the roles is not restricted in anyway, a person could use a prefix or a suffix like "dev-admin" and "prod-admin" to avoid a collision. The trade-off taken here is without this change cross-account role do not work.